examples/pnet.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Parity Technologies (UK) Ltd.
```

Comment:
I'm not a fan of having this example. If the example is supposed to demonstrate how to use a private network, then there are only 4 relevant lines in the entire example, and the rest is the unfortunate complexity of floodsub/mdns.

Reply:
Will remove or come up with something smaller.